CHANGELOG.md

```diff
@@ -29,7 +29,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 * fix: fixed filters not reset when index pattern changed ([#234](https://github.com/opensearch-project/dashboards-maps/pull/234))
 * Add custom layer visibility config to render ([#297](https://github.com/opensearch-project/dashboards-maps/pull/297))
 * Fix color picker component issue ([#305](https://github.com/opensearch-project/dashboards-maps/pull/305))
-* fix: layer filter setting been reset unexpectedly ([#327](https://github.com/opensearch-project/dashboards-maps/commits/2.x))
+* fix: layer filter setting been reset unexpectedly ([#327](https://github.com/opensearch-project/dashboards-maps/pull/327))
+* Fix data query in dashboard mode when enable around map filter ([#339](https://github.com/opensearch-project/dashboards-maps/pull/339))
```

Comment:
nit: shall we add empty line after 33?

Reply:
I found each paragraph in the existing doc don't have empty line, and it doesn’t effect the markdown view. https://github.com/opensearch-project/dashboards-maps/blob/1e3f3b386fee8a9862f9cec1e5fa9965df57a9f8/CHANGELOG.md

Reply:
Added line for each paragraph

Reply:
Thanks, the advantage will be diff will not show previous line as part of change. In this case 35 is part of your diff but going forward only the added line will show up in diff.